### ISSUE TITLE
Drop CA cert rotation option from UI

### ISSUE THREAD

**Issue by cjellick** [rancher/rancher#19052]
We have seen stability issues with rotating the CA certificates for RKE clusters.

For the initial v2.2.0 release, we want to drop that option from the cert rotation UI until we get it figured out.

This is just the "Rotate the CA and all Service certificates" option.

See:
https://github.com/rancher/rancher/issues/19050

**Comment by westlywright** [comment]
Available in ui tag `2.2.13` but also ported to master (latest2)

**Comment by davidnuzik** [comment]
Version: Master 2.2 (3/20/19) ImageID: 92dd7193e7bf (empty cache, refresh for latest UI) Single-node rancher

Confirmed issue. We have removed only the "Rotate the CA and all Service certificates" from the "Rotate Certificates" modal in the UI.

You can now _only_ either rotate an individual service or all of them. You just cannot do that and also rotate the CA now, as expected.
![image](https://user-images.githubusercontent.com/45179589/54723461-dff82c00-4b24-11e9-81ad-5bd105bfc6cc.png)

I tested individual service rotations i.e. (CLUSTERID)?action=rotateCertificates POST
When I do this I see that since there is no option int he UI caCertifications is always false, as expected:
```
caCertificates: false
services: "etcd"
```

I tested each and every individual service option (see screenshot) and finally tested with "Rotate all service certificates"
rotate all:
```
caCertificates: false
services: null
```

The server responds appropriately each time (HTTP). I also ensured our caCert string did not change for every cert rotation action in the UI (https://SERVER/v3/clusters/CLUSTERID). As such, I believe I have confirmed there are no issues in this area with the change or with cert rotation for services at least with my single-node rancher setup.

You can still rotate the CA certificate from UI API, as expected since the API still has this in.
